db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+100.times do 
+
+  Quote.create author: Faker::Name.first_name,
```

Comment:
Indentation and spacing is a little awkward in this file. `create!` would also be better than `create` here. `create` can fail to save (if validations fail) and you would have to check the returned object to know. Instead there is `create!` which will raise an exception if it wasn't successful.

I'd rather see

``` ruby
100.times do
  Quote.create!(
    author: Faker::Name.first_name,
    body:  Faker::Lorem.sentences.join(" "),
    location: Faker::Address.street_address
  )
end
```